source/material_model/ascii_reference_profile.cc

```diff
@@ -46,6 +50,22 @@ namespace aspect
       thermal_expansivity_index = profile.get_column_index_from_name("thermal_expansivity");
       specific_heat_index = profile.get_column_index_from_name("specific_heat");
       compressibility_index = profile.get_column_index_from_name("compressibility");
+
+      // these are only optional entries in the data file,
+      try
+        {
+          // read them in if they exist
+          // the strings have to be all lower case for the lookup class to find them
+          seismic_vp_index = profile.get_column_index_from_name("seismic_vp");
+          seismic_vs_index = profile.get_column_index_from_name("seismic_vs");
+          seismic_dvp_dT_index = profile.get_column_index_from_name("seismic_dvp_dt");
+          seismic_dvs_dT_index = profile.get_column_index_from_name("seismic_dvs_dt");
+        }
+      catch (...)
+        {
+          // keep the invalid unsigned int entry if the columns do not exist,
+          // we'll handle this below when we fill the outputs
```

Comment:
I added a new function to utilities that returns an invalid unsigned int if the column does not exist (individually for every column name I use it with).